neuralmonkey/config/builder.py

```diff
@@ -26,21 +26,30 @@ def create(self):
 
         class_name = class_parts[-1]
 
-        # TODO should we not assume that everything is from neuralmonkey?
-        module_name = ".".join(["neuralmonkey"] + class_parts[:-1])
-
+        simple_module_path = ".".join(class_parts[:-1])
         try:
-            module = importlib.import_module(module_name)
-        except ImportError as exc:
-            # if the problem is really importing the module
-            if exc.name == module_name:
-                raise Exception(
-                    ("Interpretation '{}' as type name, module '{}' "
-                     "does not exist. Did you mean file './{}'? \n{}")
-                    .format(self.clazz,
-                            module_name, self.clazz, exc)) from None
-            else:
-                raise
+            import tensorflow as tf
+            module = importlib.import_module(simple_module_path)
+        except ImportError:
+            try:
+                if class_parts[0] == 'tf':
+                    # Due to the architecture of TensorFlow, it must be
+                    # imported this way.
+                    tf = importlib.import_module("tensorflow")
+                    module = getattr(tf, ".".join(class_parts[1:-1]))
+                else:
+                    module_name = ".".join(["neuralmonkey"] + class_parts[:-1])
+                    module = importlib.import_module(module_name)
+            except ImportError as exc:
```

Comment:
Tuto variantu jsme s @jlibovicky resili, bohuzel to opravdu nejde, protoze importovani tf neni klasicke importovani a neexistuje napriklad nic jako tf.nn.relu ... misto toho tim jak importujes tensorflow tak ve skutecnosti importujes pouze prepisy adres kde treba zrovna to tensorflow.nn se prepise na tensorflow.python.ops.nn ... toto je nejcistsi a funkcni varianta